runtime/moonbase/src/weights/pallet_scheduler.rs

```diff
@@ -92,48 +92,47 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 	fn service_task_fetched(s: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `179 + s * (1 ±0)`
-		//  Estimated: `3644 + s * (1 ±0)`
-		// Minimum execution time: 19_742_000 picoseconds.
-		Weight::from_parts(20_300_000, 3644)
-			// Standard Error: 12
-			.saturating_add(Weight::from_parts(1_311, 0).saturating_mul(s.into()))
+		//  Estimated: `4197809`
+		// Minimum execution time: 19_388_000 picoseconds.
+		Weight::from_parts(19_704_000, 4197809)
```

Comment:
This most be reverted. This is related to broken benchmarks: https://github.com/polkadot-fellows/runtimes/pull/614